### PR TITLE
Add create proposal option to app-democracy

### LINKED ISSUE #975: Allow democracy app to submit proposals
Goal:

To make the democracy app a one-stop-shop for not only viewing, but also creation of proposals.

Background:

Currently the democracy app is read-only (and voting), we want to be able to submit proposal here instead of going through extrinsics `democracy.propose`

Pre-requisites:

Look at the type of solutions for https://github.com/polkadot-js/apps/issues/974 - here we can submit the same type of data, either as curated or anything.

Actions:

- Add actions
- Pull in same curated lists as in #974 (or create it here)

### PR DESCRIPTION
**Pull request by kwingram25**
Closes #975

Adds button and modal to submit new proposals from app-democracy instead of going through app-extrinsics.

This also includes the relocated ui-app/Extrinsic I added for #1036

### CODE REVIEW

File: packages/app-democracy/src/Propose.tsx [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+    const { isValid, accountId, method, value } = this.state;
+    const hasValue = !!value && value.gtn(0);
+
+    if (!isOpen) {
+      return null;
+    }
+
+    return (
+      <Modal
+        className='staking--Bonding'
```
[the remark this answers is not in the document]
Comment (kwingram25) [reply packages/app-democracy/src/Propose.tsx]: Yes, forgot to change it after copying :)

File: packages/app-democracy/src/Propose.tsx [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+        return api.tx.consensus.setCode;
+      } catch (error) {
+        return apiDefaultTx;
+      }
+    })();
+
+    return (
+      <>
+        <Modal.Header>
+          {t('Bonding Preferences')}
```
[the remark this answers is not in the document]
Comment (kwingram25) [reply packages/app-democracy/src/Propose.tsx]: Same

**Comment by kwingram25** [comment]
Took a stab at the help messages

**Comment by kwingram25** [comment]
Separated the overview and submission form with a redirect after submitting back to the overview

Is there a quick way to access the current app's root path? I hardcoded it as '/democracy'

**Comment by jacogr** [comment]
There is a prop passed to the app form where it is attached - basePath I believe. (Sadly I know there is at least one place where I was lazy and hardcoded it as well - but in reality should rather use that prop)

**Comment by jacogr** [comment]
Couple of conflicts (again) - at least this is the last round.

**Pull request merged**